insightfinder/README.md

```diff
@@ -21,7 +20,6 @@ Contact [Datadog support][6] or email [InsightFinder support][4].
 
 [1]: https://insightfinder.com/
 [2]: https://insightfinder.com/datadog-integration/
-[3]: https://app.insightfinder.com/auth/signup
-[4]: mailto:support@insightfinder.com
-[5]: https://docs.datadoghq.com/account_management/api-app-keys/
-[6]: https://docs.datadoghq.com/help/
+[3]: mailto:support@insightfinder.com
+[4]: https://docs.datadoghq.com/account_management/api-app-keys/
+[5]: https://docs.datadoghq.com/help/
```

Comment:
You can simply remove the `[3]: mailto:support@insightfinder.com` link and keep the other ones as they are. If you change them, you also need to update the links in the files. For instance, line 13 should be `You need a Datadog [API key and application key][4].` with this change and not `You need a Datadog [API key and application key][5].`